NNsegmentation/data.py

```diff
@@ -20,6 +20,7 @@ def load_input(file_name):
         dat = np.stack([dat[key] for key in sorted(dat.keys())], 0)
     elif os.path.splitext(file_name)[1] == '.npy':
         dat = np.load(file_name)
+    assert len(dat.shape) == 5, "Please format inputs as 5-dimensional (t, c, z, x, y) arrays"
```

Comment:
This is a relatively minor point, but numpy uses (row, col) dimension order for 2D array so are usually referred as (y, x) instead of (x, y). This doesn't change the code but just the naming in docstrings

Reply:
Got it, thanks for pointing out. I did realize sometimes the axis are flipped in cv2/numpy, this totally makes sense now.